src/notification/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { PureComponent, Fragment } from 'react'
+import { canUseDOM } from 'exenv'
+import cc from 'classcat'
+import { CrossIcon } from 'icon'
+import { createPortal } from 'react-dom'
+import { color, space } from '_utils/branding'
+import TransitionGroup from 'react-transition-group/TransitionGroup'
+import CustomTransition, { AnimationType } from 'transitions'
+import Button from 'button'
+import Text, { TextContainerType, TextDisplayType } from 'text'
+
+import style from './style'
+
+export interface NotificationProps {
+  readonly close: () => void
+  readonly isOpen: boolean
+  readonly className?: string
+}
+
+class Notification extends PureComponent<NotificationProps> {
+  private portalNode: HTMLElement
+  static defaultProps: Partial<NotificationProps> = {
+    isOpen: false,
+  }
+
+  constructor(props: NotificationProps) {
+    super(props)
+    if (canUseDOM) {
+      this.portalNode = document.createElement('div')
+      document.body.appendChild(this.portalNode)
+    }
+  }
+
+  componentWillUnmount() {
+    document.body.removeChild(this.portalNode)
+    this.portalNode = null
+  }
+
+  render() {
+    if (!canUseDOM || !this.portalNode) {
+      return null
+    }
+
+    const modalElement = (
+      <TransitionGroup component="div" className="transition-wrapper kirk-notification-container">
+        {this.props.isOpen && (
+          <CustomTransition animationName={AnimationType.SLIDE_UP}>
+            <div className={cc(['kirk-notification', this.props.className])}>
+              <Text display={TextDisplayType.TITLE} className="kirk-notification-content">
+                {this.props.children}
+              </Text>
+              <Button
+                status={Button.STATUS.UNSTYLED}
+                className="kirk-notification-cross"
+                onClick={this.props.close}
+              >
+                <CrossIcon iconColor={color.white} size="18px" />
```

Comment:
we don't have this size in the design system? Should we let it that way? @Zarthia

Reply:
We don't really have sizes for icons in the design system, only the default one